core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,33 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function blogDrafts_journey() {
```

Comment:
Functions should start with a verb. Something like deleteDraftAsBlogAdmin perhaps (more generally, "do action as actor")? You can follow a similar naming convention for the others.

Reply:
sure we can change the names accordingly